app/core/users/endpoints_users.py

```diff
@@ -826,6 +826,39 @@ async def update_current_user(
     await cruds_users.update_user(db=db, user_id=user.id, user_update=user_update)
 
 
+@router.patch(
+    "/users/fusion",
+    status_code=204,
+)
+async def fusion_users(
+    user_fusion: schemas_core.CoreUserFusionRequest,
+    db: AsyncSession = Depends(get_db),
+    user: models_core.CoreUser = Depends(is_user_a_member_of(GroupType.admin)),
+):
+    """
+    Fusion two users into one. The first user will be deleted and its data will be transferred to the second user.
+    """
+    user_kept = await cruds_users.get_user_by_email(
+        db=db,
+        email=user_fusion.user_kept_email,
+    )
+    if user_kept is None:
+        raise HTTPException(status_code=404, detail="User kept not found")
+
+    user_deleted = await cruds_users.get_user_by_email(
+        db=db,
+        email=user_fusion.user_deleted_email,
+    )
+    if user_deleted is None:
+        raise HTTPException(status_code=404, detail="User deleted not found")
+
```

Comment:
Could you log a warning with the security logger indicating that two users (with email and user id) were merged?